presto-main/src/main/java/com/facebook/presto/operator/HashBuilderOperator.java

```diff
@@ -194,6 +198,7 @@ private int getAndIncrementPartitionIndex(Lifespan lifespan)
         CLOSED
     }
 
+    private static final Logger log = Logger.get(HashBuilderOperator.class);
```

Comment:
nit: Move it to the beginning of the class (that's where we usually define logger)